ui/components/app/snaps/snap-permissions-list/snap-permissions-list.test.js

```diff
@@ -25,6 +25,7 @@ describe('Snap Permission List', () => {
   it('renders permissions list for snaps', () => {
     renderWithProvider(
       <SnapPermissionsList
+        snapId="snapid"
```

Comment:
Is this related to your changes?

Reply:
it was a fix. but will remove for now 🙏🏻

Reply:
see [1ba6cb1f7d489bea929d9b6969e964406b94543e](https://github.com/MetaMask/metamask-extension/pull/20363/commits/1ba6cb1f7d489bea929d9b6969e964406b94543e)